backend/src/utils/generateKeywords.ts

```diff
@@ -0,0 +1,17 @@
+export default function generateKeywords(name: string): string[] {
```

Comment:
while this is clever, and I think this is a cool way to assist in searchability, I think we should just defer to a fuzzy-search library such as Fuse.js (https://fusejs.io/) to handle keyword matching

Reply:
Don't libraries like this require the entire piece of data (all apts/landlords) loaded into memory?

Reply:
That's true. Honestly, since the number of landlords in Ithaca won't be big enough for it to not be loadable into memory, we can just handle it in the backend since we don't need scalability to justify Algolia. (also this is just an MVP)